dev/CommonStyles/CalendarView_themeresources_21h1.xaml

```diff
@@ -0,0 +1,789 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary 
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)"
+    xmlns:contract7NotPresent="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractNotPresent(Windows.Foundation.UniversalApiContract,7)">
```

Comment:
why file name is `dev/CommonStyles/CalendarView_themeresources_21h1.xaml`? I didn't see any contract13 or contract14 in this file.

Reply:
It really should be dev/CommonStyles/CalendarView_themeresources_21h2.xaml because it uses October Refresh CalendarView properties, but for the next couple of months we can keep it as CalendarView_themeresources_21h1.xaml until we globally change all names from *_21h1 to *_21h2.

Reply:
(By the way - none of that should be brought up publicly once I send out a brand new PR for the real thing in the next few days).